src/styles/styles.js

```diff
@@ -510,6 +510,14 @@ const styles = {
         padding: 0,
         left,
     }),
+    secureInputEyeButton: {
+        right: 0,
+        top: 0,
+        borderRadius: 6,
```

Comment:
Are these values set in stone? What happens if its set as our variables `componentBorderRadiusNormal`, etc?

Reply:
I've removed these